example/webdriverio-test/test/specs/test.e2e.ts

```diff
@@ -2,78 +2,202 @@ import {driver} from '@wdio/globals';
 import {
   CONNECT_BUTTON,
   DISCONNECT_BUTTON,
+  JOIN_BUTTON,
   SHARE_SCREEN_BUTTON,
   SWITCH_CAMERA_BUTTON,
   TOGGLE_CAMERA_BUTTON,
+  TOGGLE_MICROPHONE_BUTTON,
   TOKEN_INPUT,
   URL_INPUT,
+  VIDEO_STATISTICS_RTC,
+  AUDIO_STATISTICS_RTC,
+  TOGGLE_STATISTICS_BUTTON,
+  NO_CAMERA_VIEW,
+  VIDEO_CELL,
 } from '../../../types/ComponentLabels';
 
 import {
-  platformJellyfishToken,
+  getBasePath,
+  getElement,
+  getWebsocketUrl,
   tapApp,
   tapButton,
   typeToInput,
+  compareInputValue,
 } from '../../utils';
 
+import {
+  AddPeerRequest,
+  Configuration,
+  ConfigurationParameters,
+  PeerDetailsResponseData,
+  Room,
+  RoomApiFp,
+} from '../../../server-api';
+
 import * as assert from 'assert';
 
+const createJellysifhRoom = async () => {
+  const configParam: ConfigurationParameters = {
+    accessToken: 'development',
+    basePath: `http://${getBasePath()}:${process.env.JELLYFISH_PORT}`,
```

Comment:
Imho zmienna JELLYFISH_PORT nie powinna istniec, tylko miec 2 zmienne do url, jedna to SERVER_URL a druga DEVICE_URL, ponieważ port dla obu zmiennych zostaje ten sam

kiedy używamy testów na fizycznym urządzeniu telefon musi wiedzieć na jakie IP powinien uderzać, wtedy DEVICE_URL bedzie postaci:
adres_ip:port
kiedy używamy emu zmienna powinna być postaci
10.0.2.2:port

A SERVER_URL dla obu przypadków jest ten sam, czyli:
localhost:port lub 127.0.0.1:port